test/src/test/kotlin/graphql/nadel/tests/next/fixtures/defer/QueryContainingDeferButNonDeferredFieldIsRenamedTestSnapshot.kt

```diff
@@ -0,0 +1,106 @@
+// @formatter:off
```

Comment:
This test is correct 
(as it doesnt apply the transform directly in the `@defer`